### ISSUE TITLE
Telemetry Reuse for Jeti

### ISSUE THREAD

**Issue by mikekenyon99** [rotorflight/rotorflight-firmware#125]
Is it possible to have a similar option like the crsf_xxx_reuse for the jeti exbus so I can see headspeed on my transmitter?

Mike

**Comment by mikekenyon99** [comment]
Thats a fair comment/point :-) I can see how it would be hard to test.

**Comment by rotorflight** [comment]
Someone with Jeti and programming skills should look into this. Like Rob said, none of use is using Jeti.

**Comment by robthomson** [comment]
Do you have a working jeti + telemetry?

It may be possible to try extend this a bit with me adding sensor - and sending you a test firmware for RF?

**Comment by mikekenyon99** [comment]
Yes, currently I have a jeti tx/rx with hobbywing v4 esc.  However find the current readings very poor/inaccurate so was thinking of removing it.  However happy to try

**Comment by randomgraphics** [comment]
I have Jeti DC-24 and R6L receiver that supports telemetry, with a Hobbywing Platinum 60A V4 ESC. i'd love to help testing this feature if possible. I'm a programmer too. If someone can give me a kickstart of where to begin, i can give it a try to help implementing/improving this feature.

I read the ex-bus protocol document a bit (https://www.jetimodel.com/support/telemetry-protocol/). It looks that the exbus protocol only works in half-duplex mode that requires special wiring that mix the TX and RX signal into 1 wire.

**Comment by robthomson** [comment]
There is no need for a reuse function.

Jeti already can support multiple rpm sensors.

It's more that they just need to be tent.

Tero on discord is looking into this... Might be worth having a touch base with him to avoid duplication of effort?

**Comment by robthomson** [comment]
Closing issue as have added additional sensors - no need for reuse
